lib/rocket_pants/controller/respondable.rb

```diff
@@ -5,14 +5,17 @@ module Respondable
     SerializerWrapper = Struct.new(:serializer, :object) do
 
       def serializable_hash(options = {})
```

Comment:
This stems partly from the super loose constraint in the Gemfile

```ruby
gem 'active_model_serializers', ENV['AMS_VERSION'] || '> 0.0'
```

(I'm guessing it's a soft dependency?)